src/environment/environment.local.ts

```diff
@@ -50,5 +50,8 @@ export const environment: LocalEnvironmentFile = {
     url: 'https://dev.mes.dev-dvsacloud.uk/v1/logs/dev',
     autoSendInterval: 1000 * 60,
   },
+  user: {
+    findUser: 'https://dev.mes.dev-dvsacloud.uk/v1/users/{staffNumber}',
```

Comment:
This needs adding to config service too?

Reply:
should be in `environment.dev.ts` too I think

Reply:
Yup, its added to the config service, this is only the mock